subprojects/architecture-test/architecture-test.gradle.kts

```diff
@@ -1,7 +1,12 @@
 import gradlebuild.basics.PublicApi
 
 plugins {
-    id("gradlebuild.internal.java")
```

Comment:
This plugin only applies `gradlebuild.java-library`, acting only as an alias to that.

Reply:
I am not sure I understand why this needs to change here. Can you explain. What happens if we keep `"gradlebuild.internal.java"` here instead of applying all the plugins individually?

The idea of `gradlebuild.internal.java` that it marks all projects that are _internal_ - i.e. not part of the Gradle distribution. (Ideally all this projects should also follow the `iternal-` naming convention. Which currently only some of them do. So this one would be `internal-architecture-test`.)

Reply:
Nevermind. I did not read the PR description....

Reply:
The difference is that most projects are tested by the common CI testing setup. But a few are not. I think the real problem is in the CI setup. (Why do some projects need to be tested differently? Couldn't they at least use the same lifecycle tasks?)

Anyway, to fix this in the build we could have two plugins. Something like
`gradlebuild.internal.java`
and
`gradlebuild.internal.java-unmanaged` (don't know a good name right now) - this one is then for  "soak", "distributions-integ-tests", "architecture-test"

`gradlebuild.internal.java` would then apply the "lifecycle" plugin and the other would not.

We would also need to apply the lifecycle plugin in `distribution-module`.


If we do it like this, I think it makes sense to do all of this in one PR. What I would like to avoid is individually apply many plugins directly in a build script.